temba/orgs/models.py

```diff
@@ -705,26 +723,26 @@ def parse_location(self, location_string, level, parent=None):
         # now look up the boundary by full name
         boundary = self.find_boundary_by_name(location_string, level, parent)
 
-        if not boundary:
+        if len(boundary) == 0:
```

Comment:
Django QuerySets evaluate to None if they are empty, so these can all still be ```if boundary``` and ```if not boundary```